search-app/src/cmr/search/runner.clj

```diff
@@ -1,12 +1,13 @@
 (ns cmr.search.runner
-  (:require [cmr.search.system :as system]
-            [cmr.common.log :refer (debug info warn error)]
-            [cmr.common.config :as cfg])
-  (:gen-class))
+  "Entry point for the application. Defines a main method that accepts arguments."
+  (:require [cmr.common-app.app-events :as events]
+            [cmr.common.config :as cfg]
+            [cmr.search.system :as system]))
 
 (defn -main
   "Starts the App."
   [& args]
   (let [system (system/start (system/create-system))]
-    (info "Running...")
```

Comment:
Why remove this log event?

Reply:
I found it is redundant with another log, common-lib/src/cmr/common/system.clj

Reply:
You mean there are multiple logs with the same info? What's the text of the common-lib/src/cmr/common/system.clj log?